src/test/java/xyz/vegaone/easytracking/task/TaskServiceTest.java

```diff
@@ -111,16 +107,8 @@ public void getTaskTest() {
     @Test
     public void deleteTaskTest() {
         //given
-        UserStory userStory = new UserStory();
-        userStory.setTitle(USERSTORY_TITLE);
-        userStory.setDescription(USERSTORY_DESCRIPTION);
-        userStory.setOwner(USERSTORY_OWNER);
-        userStory.setEstimation(USERSTORY_ESTIMATION);
-        userStory.setPriority(USERSTORY_PRIORITY);
-        userStory.setProjectId(PROJECT_ID);
-        userStory.setStatus(STATUS);
 
-        UserStory savedUserStory = userStoryService.createUserStory(userStory);
+        UserStory savedUserStory = userStoryService.createUserStory(buildAndSaveUserStory());
 
         Task task = new Task();
```

Comment:
Same as the bug comment. You can safely extract this block of code to a private method.